src/components/crossDevice/CrossDeviceLink/index.js

```diff
@@ -340,32 +384,34 @@ class CrossDeviceLinkUI extends Component {
           >
             {currentViewRender()}
           </div>
-          <div role="heading" aria-level="2" className={style.styledLabel}>
-            {translate('get_link.link_divider')}
-          </div>
+          {visibleViewOptions.length > 1 && (
+            <div role="heading" aria-level="2" className={style.styledLabel}>
+              {translate('get_link.link_divider')}
+            </div>
+          )}
           <div
             className={style.viewOptionsGroup}
             aria-controls="selectedLinkView"
           >
-            {SECURE_LINK_VIEWS.filter(
-              (view) => view.id !== this.state.currentViewId
-            ).map((view) => (
-              <a
-                href="#"
-                className={classNames(
-                  theme.link,
-                  style.viewOption,
-                  style[view.className]
-                )}
-                ref={(node) => (this.viewOptionBtn = node)}
-                onClick={preventDefaultOnClick(() =>
-                  this.handleViewOptionSelect(view.id)
-                )}
-                key={`view_${view.id}`}
-              >
-                {translate(view.label)}
-              </a>
-            ))}
+            {visibleViewOptions
+              .filter((view) => view.id !== currentViewId)
+              .map((view) => (
+                <a
+                  href="#"
+                  className={classNames(
+                    theme.link,
+                    style.viewOption,
+                    style[view.className]
+                  )}
+                  ref={(node) => (this.viewOptionBtn = node)}
+                  onClick={preventDefaultOnClick(() =>
+                    this.handleViewOptionSelect(view.id)
+                  )}
+                  key={`view_${view.id}`}
+                >
+                  {translate(view.label)}
+                </a>
+              ))}
```

Comment:
I'm curious what'll be rendered if 2 options were provided (1 visible, 1 other). Does that break the view or display as normal?

Reply:
The view will still display as normal, but end user will only see the 1 other option under the `-- or choose an alternative method --` divider.